modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-migration-uyuni.adoc

```diff
@@ -3,18 +3,18 @@ ifeval::[{suma-content} == true]
 :noindex:
 endif::[]
 
-In {productname} 2024.04, the containerized proxy is managed by a set of systemd services.
+In {productname} {productnumber}, the containerized proxy is managed by a set of systemd services.
 
 In {productname} {productnumber}, management of the containerized proxy was re-designed and made simpler with the [command]``mgrpxy`` tool.
 
 This section will help you migrate from the legacy [systemitem]``systemd`` proxy using the new [command]``mgrpxy`` tool.
 
 [IMPORTANT]
 ====
-An in-place migration from {productname} 2024.04 to 2024.06 will remain unsupported due to the HostOS change from {sles} 15 SP4 to {leapmicro} {microversion}
+An in-place migration from previous releases of {productname}  to {productnumber} will remain unsupported due to the HostOS change from {sles} 15 SP4 to {leapmicro} {microversion}
```

Comment:
Probably meant "from opensuse leap to {leapmicro}"

Reply:
Fixed.